internal/project/common/repository.go

```diff
@@ -19,36 +19,28 @@ import (
 )
 
 // Repository sets up repository settings.
-type Repository struct { //nolint:govet,maligned
+type Repository struct {
+	DeprecatedEnableLicense *bool `yaml:"enableLicense"`
```

Comment:
I hate when fields are ordered for GC, but they make little sense... I would rather keep them order by some sense

Reply:
fixed